scripts/build_canonical_gliner_dataset.py

```
@@ -0,0 +1,868 @@
+        path_in_repo="build_report.json",
+        repo_id=repo_id,
+        repo_type="dataset",
+    )
+    print(f"[info] Uploaded README.md and build_report.json to {repo_id}")
+
+
+def default_repo_ids() -> tuple[str, str]:
+    info = whoami()
+    username = info.get("name") or info.get("fullname")
+    if not username:
+        raise RuntimeError("Could not resolve Hugging Face username from whoami().")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)